sdk/messaging/internal/cbs/cbs.go

```diff
@@ -1,90 +0,0 @@
-// Package cbs provides the functionality for negotiating claims-based security over AMQP for use in Azure Service Bus
```

Comment:
What will this mean for Event Hubs?

Reply:
When we get there I'll merge whatever we end up with back into internal.

Reply:
(I have some more changes coming for this but I didn't want to do it all in the same PR)